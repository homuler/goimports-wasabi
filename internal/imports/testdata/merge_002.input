// merge multiple single import declarations (grouping)
package main

import "github.com/homuler/foo"
import "context"
import "local/baz"
import "fmt"
import "github.com/homuler/bar"
import "local/qux"

var (
	x = foo.X
	y = bar.Y
	w = baz.W
	z = qux.Z
)

func main() {
	fmt.Println(context.Background())
	fmt.Printf("%v %v %v %v\n", x, y, w, z)
}
