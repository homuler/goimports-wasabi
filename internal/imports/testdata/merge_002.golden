// merge multiple single import declarations (grouping)
package main

import (
	"context"
	"fmt"

	"github.com/homuler/bar"
	"github.com/homuler/foo"

	"local/baz"
	"local/qux"
)

var (
	x = foo.X
	y = bar.Y
	w = baz.W
	z = qux.Z
)

func main() {
	fmt.Println(context.Background())
	fmt.Printf("%v %v %v %v\n", x, y, w, z)
}
